Game/buildings.gd

```diff
@@ -0,0 +1,21 @@
+extends Node
+
+@export var _buildings_dict: Dictionary[Types.Building, BuildingResource]
+
+func get_icon(building_type: Types.Building) -> AtlasTexture:
```

Comment:
I'm not a fan of creating accessor methods when we can just pull these values directly off the resource object itself. Why not just create a method to return the building resource and then pull those values off there?